### PR TITLE
Add .travis.yml. Now only running flake

### PR DESCRIPTION
**Pull request by wazaundtechnik**
Title should be self-explanatory

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+sudo: false
```
Comment (kemskems) [inline .travis.yml]: What's this for? Does Travis by default run everything as root?

**Comment by kemskems** [comment]
LGTM. Minor inline question.

I have a wishlist tho. I think it's nice if Travis can send notification to our Telegram chat on build completion :-)

**Comment by wazaundtechnik** [comment]
> I have a wishlist tho. I think it's nice if Travis can send notification to our Telegram chat on build completion :-)

Thanks! Issue created.

File: .travis.yml (hunk 1 above)
Comment (wazaundtechnik) [inline .travis.yml]: Explicit is better than "i accept whatever".

**Pull request merged**